centralstore/redis_store.go

```diff
@@ -646,6 +646,9 @@ func (t *tracesStore) addStatuses(ctx context.Context, conn redis.Conn, cspans [
 
 	commands := make([]redis.Command, 0, 3*len(cspans))
 	for _, span := range cspans {
+		if span.SpanID == "" {
```

Comment:
This is to prevent we store the fake root span centra store sent because that's how we differentiate a signal span vs an actual span. I will add a comment here for this